GVRf/Sample/gvrtextviewsample/src/org/gearvrf/gvrtextviewsample/SampleViewManager.java

```diff
@@ -31,7 +31,7 @@
     GVRConsole console;
     SampleActivity mActivity;
     boolean init = false;
-    private final String[] strs = new String[] { "good", "verygood",
+    private final String[] strings = new String[] { "good", "verygood",
             "veryverygood", "veryverygood", "veryveryverygood" };
     private final int[] colors = new int[] { Color.BLUE, Color.BLACK,
```

Comment:
is there something we can do about the colors?  i never see the blue or black text.